Sources/Jaeger/Jaeger/Agent/CoreData/AgentCoreDataObjectModel.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  AgentCoreDataObjectModel.swift
+//  Jaeger
+//
+//  Created by Simon-Pierre Roy on 2/1/19.
+//
+
+import CoreData
+
+/// The Core Data model for the Agent written in Code.
+let agentCoreDataObjectModel: NSManagedObjectModel = {
```

Comment:
Since this is a global constant, can we wrap it in something to make like something other than a stray variable? Something like:

```swift
enum AgentCoreDataObjectModel {
    static let model: NSManagedObjectModel = { ... }()
}
```

Reply:
Done!